### ISSUE TITLE
[E02, S05] As a user I want to be able to browse as a guest.

### ISSUE THREAD

**Issue by radinat** [EnterDAO/ProjectManagement#110]

**Comment by entrxyz** [comment]
@radinat if you want to store games/events in Favourites, shouldn't they login first? Maybe when doing such action, to display a pop-up or something and ask them to log-in with existing wallet or create a new one, etc. They should be able to close the pop-up and continue to browse as a guest. Can you please check this with George and most probably we'll have to create specific design for this.

**Comment by radinat** [comment]
Not necessarily, guests will be able to add games/events to favorites and this will be stored locally. But they won't be able to Install/Join them. They will still see a pop-up that prompts them to log in every time they load a page.

**Comment by radinat** [comment]
I think we need to discuss the first point with the devs, but I believe the best case would be to not lose info when closing the app and the favourites to be associated with your wallet once you create an account. 
Then if you log in with another wallet, you can access the favourites associated with that wallet.
